README.md

```diff
@@ -349,21 +348,6 @@ const Counter = flowMax(
       <button onClick={decrement}>-</button>
     </>
 )
-
-const IncrementByX = flowMax(
-  addStateHandlers(
-    {count: 0},
-    {
-      increment: ({count}, {x}) => () => ({count: count + x}),
-    },
-    ['x']
-  ),
-  ({count, increment}) =>
-    <>
-      Count: {count}
-      <ExpensiveToRender onClick={increment} />
-    </>
-)
```

Comment:
Why remove the example?

Reply:
This was a second example that was just there to illustrate the dependencies prop